ingest/expression_files/expression_files.py

```diff
@@ -16,17 +16,16 @@
 
 from bson.objectid import ObjectId
 from mypy_extensions import TypedDict
-from pymongo.errors import BulkWriteError
 
 try:
     from ingest_files import DataArray
     from monitor import setup_logger
-    from mongo_connection import MongoConnection
+    from mongo_connection import MongoConnection, graceful_auto_reconnect
 except ImportError:
     # Used when importing as external package, e.g. imports in single_cell_portal code
     from ..ingest_files import DataArray
-    from ..monitor import setup_logger
-    from ..mongo_connection import MongoConnection
+    from ..monitor import setup_logger, graceful_auto_reconnect
```

Comment:
I can't see why an import of graceful_auto_reconnect was added from monitor - is this needed?

Reply:
8ceccfc2

Reply:
Thanks for catching this!